test/libsolidity/semanticTests/various/erc20.sol

```diff
@@ -94,14 +94,37 @@ contract ERC20 {
     }
 }
 // ====
-// compileViaYul: true
+// compileViaYul: also
 // ----
+// constructor()
+// ~ emit Transfer(address,address,uint256): #0x00, #0x1212121212121212121212121212120000000012, 0x14
+// gas irOptimized: 23446
+// gas legacy: 833310
```

Comment:
Curious: what is being generated here?! 👀

Reply:
Actually, the other numbers here seem to be wrong :( I'l check why.